css/templates/lune/page-vars.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --content-max-width: 100%;
+
+  --z1: 1; /* body */
+  --z2: 2; /* overlay */
+  --z3: 3; /* sidebar mask */
+  --z4: 4; /* sidebar */
+  --z5: 5; /* header */
+
+  --layout-header-height-mobile: 4.375rem; /* 70px */
+  --layout-header-height-desktop: 5.75rem; /* 92px */
+
+
+
+  /* Type */
+  --font-family: "Lato", -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", Arial, sans-serif;
+  --font-weight-black: 900;
+
+  /* Responsive headings type scale */
+  --h00: 6.875rem; /* 110px */
```

Comment:
Following the conversation we had yesterday, we have had a chat with the designer of the template, and cut down the amount of variations in font sizes. We'll be using the basscss responsive type scale add. We've had to add --h7 as we're one over